src/custom/state/price/reducer.ts

```diff
@@ -56,7 +57,8 @@ function initializeState(
 function getResetPrice(sellToken: string, buyToken: string, kind: string) {
   return {
     amount: null,
-    token: kind ? sellToken : buyToken
+    // When we buy, the price estimation is given in sell tokens (if we sell, we give it in sell tokens)
+    token: kind === OrderKind.BUY ? sellToken : buyToken
```

Comment:
the comment is confusing.

but this is: for buy orders, we want to reset the price for the sellToken, and for sell orders we want to reset the price for the buyToken

right?

Reply:
I second this.

Reply:
done 👌